Libraries/Image/RCTImageLoader.m

```diff
@@ -417,8 +420,16 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
 
     // Download image
     RCTNetworkTask *task = [_bridge.networking networkTaskWithRequest:request completionBlock:^(NSURLResponse *response, NSData *data, NSError *error) {
-      if (error) {
-        completionHandler(error, nil);
+      if (error || !response || !data) {
+        NSError *someError = nil;
+        if (error) {
+          someError = error;
+        } else if (!response) {
```

Comment:
The defense code around 366 is for `processResponse`, the defense code around 427 is for `RCTNetworkTask`'s code.
The `processResponse` is also used outside  `RCTNetworkTask`'s code.